src/openloco/openloco.cpp

```diff
@@ -440,6 +444,38 @@ namespace openloco
         }
     }
 
+    // 0x0043D9D4
+    static void editor_tick()
+    {
+        if (!is_editor_mode())
+            return;
+
+        switch (_editorStep)
+        {
+            case 0:
+                if (WindowManager::find(WindowType::objectSelection) == nullptr)
+                    windows::ObjectSelectionWindow::open();
+                break;
+
+            case 1:
+                // Scenario/landscape loaded?
+                if ((addr<0x00525E28, uint32_t>() & 1) != 0)
```

Comment:
Any reason why this isn't a `loco_global`

Reply:
It's being used like this in a few other files. The original idea was to have a `loco_global` for a particular address in one file only, using accessors for the rest, right? We've strayed a bit from that, but I thought it best to keep the number of `loco_global`s to a minimum.